### ISSUE TITLE
Compare Numeric with HF Model

### ISSUE THREAD

**Issue by omkar-334** [keras-team/keras-hub#2048]
Hi, 
I wanted to find issues/todo's to contribute to Keras, so I searched for TODO in the codebase.    
I found this specific todo appearing about 10 times  - mostly in this directory `keras_hub\src\utils\transformers` .   

`
TODO: compare numerics with huggingface model
`
What does this mean? Comparing the generated tokens/logits?  
Thank you

**Comment by omkar-334** [comment]
@mattdangerw @SamanehSaadat @divyashreepathihalli

**Comment by divyashreepathihalli** [comment]
hi @omkar-334  we are missing built in transformers/timm, built in converters for the following models- please go ahead and add them if you are interested.
bart
bloom
clip
csp darknet
debertav3
f_net
falcon
llama (2)
mit
mobilenet
opt
phi3
roberta
vgg (missing tests only)
xlm-roberta

**Comment by pandeyshub-byte** [comment]
Hey @omkar-334,
It means comparing the output logits/hidden states (not the text tokens) between HF and Keras-hub to make sure they match numerically (usually within 1e-4).
We've listed out most of these converters in #2983 , you may look into these and feel free to contribute.
